pkg/security/tests/latency_test.go

```diff
@@ -0,0 +1,82 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build stresstests
+// +build stresstests
+
+package tests
+
+import (
+	"flag"
+	"fmt"
+	"os/exec"
+	"testing"
+
+	"github.com/DataDog/datadog-agent/pkg/security/secl/rules"
+)
+
+var (
+	coreID  int
+	nbRuns  int
+	nbSkips int
+	host    string
+)
+
+// bench induced latency for DNS req
+func benchLatencyDNS(t *testing.T, rule *rules.RuleDefinition, executable string) {
+	var ruleDefs []*rules.RuleDefinition
```

Comment:
It could be inside the following if statement ?

Reply:
Yep! Updated on last commit.